lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -214,4 +223,112 @@ void LLDBMemoryReader::popLocalBuffer() {
   m_local_buffer_size = 0;
 }
 
+std::pair<uint64_t, uint64_t>
+LLDBMemoryReader::addModuleToAddressMap(ModuleSP module) {
+  // We use 0x2000000000000000 to tell that the address is one mapped by us when
+  // doing memory reads later on. This should not clash with any masks used by
+  // Swift.
+  uint64_t next_available_address = 0x2000000000000000;
+  if (!m_address_map.empty()) {
+    // We map the images contiguously one after the other, all with the highest
+    // bit set.
+    next_available_address = m_address_map.back().first;
+    // Add 8 to it to keep it pointer aligned. This assumes the previous address
+    // was also pointer aligned.
```

Comment:
As it's written, this comment doesn't really make sense, because this only works if the address was already aligned and then adding 8 doesn't change anything in terms of aligned-ness. SO either the comment is wrong or this should be a call to `llvm::alignTo()` from `MathExtras.h`.

Reply:
It should work as the size of the previous image is aligned, I wasn't aware of `llvm::alignTo()` though, I'll use that instead.

Reply:
I changed the bit manipulation further down to use `llvm::alignTo()`, but decided to keep the plus 8 here, as it's actually clearer to understand, but changed the code and comments to clarify that we add 8 to the end of the previous image's address (which is already pointer aligned).